### PR TITLE
Lasttemplate

### PR DESCRIPTION
**Pull request by J2ZROMERO**
> During the project:
- Add .eslintrc.json
- Add the "click" event to the menu bar (hamburger icon).
- Modify display from mobile.
- Add the "click" event to the "X" icon when the display appears.
- Add the "click" event to "Portfolio" and scroll event when is clicked
- Add the "click" event to "About" and scroll event when is clicked
- Add the "click" event to "Contact" and scroll event when is clicked

### CODE REVIEW

File: main_view.html [hunk 1]
```diff
@@ -7,36 +7,33 @@
...
+        <img class="botton_line" src="images/Shape.svg" alt="">
+            </nav>
+    <header class="header_mobile">
+        <div class="menu">
+            <a href="#" class="logo"> Welcome </a>
+            <nav class="menu_img">
+                <img  class="menu_hamburguer" src="images/Enabledmenu_bar.svg" alt="">
+            </nav>
+        </div>
+    </header>
```
Comment (gemmen29) [inline main_view.html]: To match the Figma design:
- [ ] Kindly choose the right color and size of the close button.
- [ ] Please change the color of the `Portfolio` link to the right color as you did with the other options

| Your design | Figma design |
| --- | --- | 
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @J2ZROMERO, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by J2ZROMERO** [comment]
- Apply styles "X"  icon.
- Apply styles to the portfolio.

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @J2ZROMERO, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] All linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**